src/tpl/layouts/default.nunj

```diff
@@ -41,6 +41,8 @@
     {% endif %}
 </head>
 <body>
+    {% include "../partials/cookie-law.nunj" %}
```

Comment:
I don’t like partials here. It's time to remove it. It tends to use structure folder as container of all partials. I would suggest `tpl/component(s)/cookie-law.nunj`.